src/runtime_src/core/tools/xbutil2/SubCmdScan.cpp

```diff
@@ -27,22 +27,35 @@ namespace po = boost::program_options;
 // System - Include Files
 #include <iostream>
 
+// ======= R E G I S T E R   T H E   S U B C O M M A N D ======================
+#include "SubCmd.h"
+static const unsigned int registerResult = 
+                    register_subcommand("scan", 
```

Comment:
in xbutil, list is the same as scan and i actually removed "list" from help message and only left scan. maybe we should do the same for xbutil2 as well?

Reply:
Great to know!  Will make those changes as well.